src/sqlite3gen.cpp

```diff
@@ -107,7 +175,7 @@ const char * schema_queries[][2] = {
       "\tstatic               INTEGER DEFAULT 0, -- 0:no 1:yes\n"
       "\tconst                INTEGER DEFAULT 0, -- 0:no 1:yes\n"
       "\texplicit             INTEGER DEFAULT 0, -- 0:no 1:yes\n"
-      "\tinline               INTEGER DEFAULT 0, -- 0:no 1:yes\n"
+      "\tinline               INTEGER DEFAULT 0, -- 0:no 1:yes 2:both (set after encountering inline and not-inline)\n"
```

Comment:
'both' is a local neologism so that the generator can know when it has seen both parts of a memberdef with separate declaration and definition.